setup.py

```diff
@@ -10,6 +15,23 @@
     'complete': open('complete-requirements.txt').readlines()
 }
 
+class PreSDistNLTKDataUnpackCommand(sdist):
+    """Before creating source distribution, untar nltk data files, so they'll be included via the manifest file.
+
+    Note the usage of `pkg_resources.resource_filename` for pathing assumes you're building the source distribution with your cwd
+    in the repo.
+    """
+    def run(self):
+        nltk_data_tarball_path = pkg_resources.resource_filename('nlp_primitives', str(pathlib.Path('data','nltk-data.tar.gz')))
+        nltk_data_extract_path = pkg_resources.resource_filename('nlp_primitives', str(pathlib.Path('data')))
+        print(f'Extracting nltk data files from {nltk_data_tarball_path} to {nltk_data_extract_path}')
+        print(pkg_resources.resource_listdir('nlp_primitives', str(pathlib.Path('data'))))
+        tar = tarfile.open(nltk_data_tarball_path, "r:gz")
+        tar.extractall(path=nltk_data_extract_path)
+        tar.close()
+        print(f'Extraction of nltk data files complete')
+        sdist.run(self)
```

Comment:
It would be great if someone could read this carefully. @rwedge @thehomebrewnerd @ctduffy @eccabay 

Based on my testing and reading of `pkg_resources`, I believe `pkg_resources` will always point to a directory in which the current user has readwrite access.